src/NUnitFramework/framework/Constraints/CollectionEquivalentConstraint.cs

```diff
@@ -34,6 +34,12 @@ public class CollectionEquivalentConstraint : CollectionItemsEqualConstraint
         private readonly IEnumerable _expected;
 
         /// <summary>
+        /// The <see cref="CollectionTally"/> of the last performed collection equivalence
+        /// comparison.
+        /// </summary>
+        private CollectionTally _lastPerformedTally;
```

Comment:
This hasn't changed. I still don't like it. Are we performing more than one tally?